### PR TITLE
Feature/DS-100 Upload components

### PR DESCRIPTION
**Pull request by kamilmateusiak**
- ProgressBar
- ProgressCircle
- FileUploadProgress (ProgressBar with title, icon, etc.)
- UploadBar - complete component for upload of single or multiple files
progress

A big number of added lines is mostly due to complicated docs. Don't be afraid!:)

### CODE REVIEW

File: src/components/Progress/ProgressBar.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import styles from './style.scss';
```
Comment (marcinsawicki) [inline src/components/Progress/ProgressBar.js]: missing space

Comment (kamilmateusiak) [reply src/components/Progress/ProgressBar.js]: we didn't decide to use this convention in the ds

**Comment by quarties** [comment]
I'll make a review after the changes we discussed yesterday during the meeting.

**Comment by kamilmateusiak** [comment]
@quarties done, please take a look ;)

**Pull request merged**